src/AppInstallerSharedLib/Public/AppInstallerErrors.h

```diff
@@ -138,6 +138,10 @@
 #define APPINSTALLER_CLI_ERROR_AUTHENTICATION_INTERACTIVE_REQUIRED  ((HRESULT)0x8A150076)
 #define APPINSTALLER_CLI_ERROR_AUTHENTICATION_CANCELLED_BY_USER     ((HRESULT)0x8A150077)
 #define APPINSTALLER_CLI_ERROR_AUTHENTICATION_INCORRECT_ACCOUNT     ((HRESULT)0x8A150078)
+#define APPINSTALLER_CLI_ERROR_NO_REPAIR_INFO_FOUND                 ((HRESULT)0x8A150079)
+#define APPINSTALLER_CLI_ERROR_REPAIR_NOT_APPLICABLE                ((HRESULT)0x8A15007A)
+#define APPINSTALLER_CLI_ERROR_EXEC_REPAIR_FAILED                   ((HRESULT)0x8A15007B)
+#define APPINSTALLER_CLI_ERROR_REPAIR_NOT_SUPPORTED                 ((HRESULT)0x8A15007C)

```

Comment:
Please also update the error codes doc at https://github.com/microsoft/winget-cli/blob/master/doc/windows/package-manager/winget/returnCodes.md